src/DependencyManagement/DependencyManager.cs

```diff
@@ -93,17 +100,28 @@ internal void ProcessDependencyDownload(MessagingStream msgStream, StreamingMess
             {
                 var rpcLogger = new RpcLogger(msgStream);
                 rpcLogger.SetContext(request.RequestId, null);
-                if (Dependencies.Count == 0)
+
+                if (!_shouldUpdateFunctionAppDependencies)
                 {
-                    // If there are no dependencies to install, log and return.
-                    rpcLogger.Log(LogLevel.Trace, PowerShellWorkerStrings.FunctionAppDoesNotHaveDependentModulesToInstall, isUserLog: true);
-                    return;
+                    if (!string.IsNullOrEmpty(_dependenciesNotUpdatedMessage))
+                    {
+                        // We were not able to update the function app dependencies.
+                        // However, there is a previous installation, so continue with the function app execution.
+                        rpcLogger.Log(LogLevel.Warning, _dependenciesNotUpdatedMessage, isUserLog: true);
+                        return;
```

Comment:
Consider returning after the end of "else", so that it's clear that you will return regardless of the _dependenciesNotUpdatedMessage value.

Reply:
That's a good point. It makes it easier to read. Fixed.